File: 1.js
function sumEvenElements(arr) {
    let sumEven = 0;
    arr.forEach(num => {
        if (num % 2 === 0) {
            sumEven += num;
        }
    });
    return sumEven;
}
// Найти сумму четных элементов массива on javascript
// Пример использования
const array = [1, 2, 3, 4, 5, 6];
const result = sumEvenElements(array);
console.log(result);

File: 1.py
def sum_even_elements(arr):
    sum_even = 0
    for num in arr:
        if num % 2 == 0:
            sum_even += num
    return sum_even

# Пример использования
array = [1, 2, 3, 4, 5, 6]
result = sum_even_elements(array)
print(result)
